### DISCUSSION TITLE
Weak Aura Skins not Skinned the Same

### DISCUSSION THREAD

**Discussion by Theunwow** [SFX-WoW/Masque_Caith#28] [Questions] [answered]
![image](https://user-images.githubusercontent.com/44935842/48797285-c0183f00-eccf-11e8-946f-95ed4cd8d863.png)
As you can see in the image, the top Weak Auras (Unleash Life and Spirit Walker's Grace) have a 1px border on the outside of them, but all of the bottom auras are missing that 1px border. All weak auras are made exactly the same way, and all have the exact same skin settings - I don't know how to get the 1px border to show on the bottom weak auras.

**Comment by StormFX** [comment, marked as answer]
Masque skins all buttons the same so all I can suggest is adjusting the position by a few pixels. The game isn't very good at rendering at pixel accuracy, especially when the frames are created differently.
